management_api_app/service_bus/deployment_status_update.py

```diff
@@ -74,8 +74,26 @@ def create_updated_deployment_document(resource: dict, message: DeploymentStatus
     resource["deployment"]["status"] = message.status
     resource["deployment"]["message"] = message.message
 
+    # although outputs are likely to be relevant when resources are moving to "deployed" status,
+    # lets not limit when we update them and have the resource process make that decision.
+    output_dict = {}
+    for output in message.outputs:
+        output_dict[output.Name] = dequote(output.Value)
+
+    original_params_dict = dict(resource["resourceTemplateParameters"])
+    resource["resourceTemplateParameters"] = {**original_params_dict, **output_dict}
+
     return resource
 
+def dequote(s):
```

Comment:
does string.strip("'").strip('"') not work?

Reply:
It's almost the same, although using strip doesn't check that the start and end of the string are the same.
I don't know what will produce this string but: 
word" 
will become:
word

I can change that.

Reply:
It won't produce the same results every time, but maybe that's ok...
Consider this string: word" (just 1 quote) 
Your process will remove that last quote despite it not having a matching open one.

Anyway, I changed the code with your suggestion.